site/content/docs/5.1/migration.md

```diff
@@ -66,28 +90,13 @@ Your custom Bootstrap CSS builds should now look something like this with a sepa
 
 ### Additional changes
 
-- **Introduced new `$enable-container-classes` option. —** Now when opting into the experimental CSS Grid layout, `.container-*` classes will still be compiled, unless this option is set to `false`.
-
-- **Alerts, badges, buttons, dropdowns, navbars, pagination, popovers, and tooltips are now built with CSS variables. —** While Sass still underpins everything, each of these components have been updated to include several CSS variables on the component base classes (e.g., `.btn`), allowing for more real-time customization of Bootstrap.
-
-  Read more about CSS variables in each component on their respective documentation pages.
+- **Introduced new `$enable-container-classes` option. —** Now when opting into the experimental CSS Grid layout, `.container-*` classes will still be compiled, unless this option is set to `false`. Containers also now keep their gutter values.
 
 - **Thicker table dividers are now opt-in. —** We've removed the thicker and more difficult to override border between table groups and moved it to an optional class you can apply, `.table-group-divider`. [See the table docs for an example.]({{< docsref "/content/tables#table-group-dividers" >}})
 
-- **Updated several components to use CSS variables. —** While Sass still underpins everything, each of these components have been updated to include several CSS variables on the component base classes (e.g., `.btn`), allowing for more real-time customization of Bootstrap. The following components are now built with CSS variables:
-
-  - [Alerts]({{< docsref "/components/alerts/" >}})
-  - [Badges]({{< docsref "/components/badge/" >}})
-  - [Breadcrumbs]({{< docsref "/components/breadcrumb/" >}})
-  - [Buttons]({{< docsref "/components/buttons/" >}})
-  - [Dropdowns]({{< docsref "/components/dropdowns/" >}})
-  - [Navbars]({{< docsref "/components/navbar/" >}})
-  - [Popovers]({{< docsref "/components/popovers/" >}})
-  - [Tooltips]({{< docsref "/components/tooltips/" >}})
-
-- **Alerts, badges, breadcrumbs, buttons, navbars, popovers, and tooltips are now built with CSS variables. —** While Sass still underpins everything, each of these components have been updated to include several CSS variables on the component base classes (e.g., `.btn`), allowing for more real-time customization of Bootstrap.
+- Added `.form-check-reverse` modifier to flip the order of labels and associated checkboxes/radios.
 
-- Updated our grid systems so that containers now keep their gutter values, and enabling the experimental CSS Grid no longer removes container classes.
+- Added striped columns support to tables via
```

Comment:
@mdo Something's missing here, the sentence is not complete (see https://twbs-bootstrap.netlify.app/docs/5.1/migration/#additional-changes). Do you want me to create an issue to track it?